symbols/man_made/manhole.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg height="4" width="4">
+  <circle cx="2" cy="2" r="2" fill="black" />
```

Comment:
Current dot renderings (shop=* and others) does not need an svg file but uses native mapnik circles.